enhancements/network/system-networkpolicies.md

```diff
@@ -0,0 +1,386 @@
+---
+title: Network Policies for System Namespaces
+authors:
+  - "@danwinship"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-01-18
+last-updated: 2021-01-18
+status: provisional
+---
+
+# Network Policies for System Namespaces
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+An OCP cluster contains various services that should not be accessible
+to unprivileged users.
+
+Currently, authorization for these services is handled via TLS
+certificates; unauthorized users can _connect_ to the service, but
+they cannot _authenticate_ to it, and so the service is protected.
```

Comment:
if you want to be pedantic, they can _authenticate_, they're just not _authorized_ to access the resource.